pom.xml

```diff
@@ -17,8 +17,8 @@
     </properties>
     <modules>
         <module>user-org-util</module>
-	<module>actor-service</module>
-	<module>user-org-actor</module>
+	    <module>actor-service</module>
```

Comment:
format it please...been noticing we only format java files in precommit.

Reply:
fixed.